src/test/groovy/net/fabricmc/loom/BuildUtils.groovy

```diff
@@ -0,0 +1,213 @@
+package net.fabricmc.loom
```

Comment:
any reason doing this over than having them as normal files and reading them in?

Reply:
Having files in the source directory would cause them to get built, which isn't desirable. This way, their contents may be generated in a temporary directory and "fresh" builds can be run as if the plugin was being consumed normally.

They could be separated into different files and trimmed down, but I pretty much just copied the current example mod as a base.

Reply:
Humm ok, I guess they wont change a lot and they dont seem to hard to maintain so thats fine.